src/OpenLoco/src/Objects/AirportObject.cpp

```diff
@@ -60,7 +60,9 @@ namespace OpenLoco
             buildingVariationParts[i] = reinterpret_cast<const uint8_t*>(remainingData.data());
             auto* ptr = buildingVariationParts[i];
             while (*ptr++ != 0xFF)
+            {
                 ;
+            }
```

Comment:
This change seems unrelated?

Reply:
I did a whole project clang format as it was easiest way to format. Looks like these files haven't been touched since the change to always have {}